common/views/components/PrismicImage/index.tsx

```diff
@@ -28,20 +28,30 @@ export type Props = {
   desaturate?: boolean;
 };
 
+/**
+ * Converts a rem string to pixels
+ * Assumes 1rem = 16px (browser default)
+ */
+function remToPx(remValue: string): number {
+  const remNumeric = parseFloat(remValue);
+  return remNumeric * 16;
+}
+
 export function convertBreakpointSizesToSizes(
```

Comment:
This function confuses me and I _think_ there might have been developments with the img sizes attribute that mean we could simplify it. Even if there haven't been any developments and I'm imagining it, I think/hope we can tidy it up so that it's clearer what it is doing